ambry-named-mysql/src/integration-test/java/com/github/ambry/named/MySqlNamedBlobDbIntegrationTest.java

```diff
@@ -210,6 +230,79 @@ public void testExpiredBlobs() throws Exception {
         namedBlobDb.get(account.getName(), container.getName(), blobName).get());
   }
 
+  /**
+   * Test behavior with stale blob cleanup
+   */
+  @Test
+  public void testCleanupStaleBlobs() throws Exception {
+    Calendar calendar = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
+    calendar.add(Calendar.MONTH, -2);
+    long twoMonthAgo = calendar.getTimeInMillis();
+    long twoMonthAgoABitLater = twoMonthAgo + 1;
```

Comment:
please add more time to this. 1 millisecond is too short sometimes. could you make it 1 minute.

Reply:
That's the beauty of math, here 1 millisecond is enough. I updated at other places when needed and renamed the variables